composer.json

```diff
@@ -24,11 +24,12 @@
         }
     ],
     "require": {
-        "php": "^7.0"
```

Comment:
We still need to support PHP 7 here. Following the core library, let's go for `"php": "^7.3|^8.0"` here 👍